### PR TITLE
feat: Add blob topic handler

### PR DESCRIPTION
**Pull request by 0x00101010**
### Description

Add blob topic handler to decode blob p2p message and produce TraceEvent.

```
info  | 17:51:25.943235 | Handling blob gossip message              
PeerID=16Uiu2HAmTA28PLzU7oYBQujqRWN8oG8wBmgF6zhUe99HzWuU7GyV 
RemotePeerID=16Uiu2HAmAcLyYnyjgrznMaCXwRUfWJGChVyr2Qdz3cr6sKwT2Hvn 
MsgID=0e0002377b3edbb39a1c486e02fb421962f7466b 
MsgSize=9598 
Topic=/eth2/d31f6191/blob_sidecar_2/ssz_snappy 
Seq=[] 
Slot=5005157 
ValIdx=260 
TimeInSlot=1.943217312 
StateRoot=0xab69b4b0e8904205840bafaeca266a86c78eeb16a9eee13aa5cf2a9db18747bd 
BodyRoot=0x718cb2446668e05c3fe63c326902a7d656b72635e277b2f6266d134a53ce6a6c 
ParentRoot=0xf27c0e927ac5670184eec5ebdf05740997d0cfd56d643b26e368c84b7e547b4f
```

### CODE REVIEW

**Review by Savid — APPROVED** [review]

File: eth/pubsub.go [hunk 1]
```diff
@@ -165,6 +168,65 @@ func (p *PubSub) handleBeaconBlock(ctx context.Context, msg *pubsub.Message) err
...
+			"MsgSize", len(msg.Data),
+			"Topic", msg.GetTopic(),
+			"Seq", msg.GetSeqno(),
+			"Slot", slot,
+			"ValIdx", proposerIndex,
+			"TimeInSlot", now.Sub(slotStart).Seconds(),
+			"StateRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetStateRoot()),
+			"BodyRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetBodyRoot()),
+			"ParentRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetParentRoot()),
+		)
```
Comment (guillaumemichel) [inline eth/pubsub.go:218]: I don't understand why the `slog` logging is required, this information is duplicate with the `evt`.

Comment (0x00101010) [reply eth/pubsub.go]: It's not necessarily "required", but helps a little in terms of debugging and showing what's coming from those topics.

Removed it for now, and if we need them in the future, we can add back with Debug or Trace level logging.

**Review by cortze — APPROVED** [review]
LGTM! many thanks @0x00101010 for the PR!

**Review by guillaumemichel — APPROVED** [review]
Thanks @0x00101010 for your contribution!

**Pull request merged**